src/utils/components/HardwareDevices/HardwareDevicesPageTable.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from 'react';
+
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { VirtualizedTable } from '@openshift-console/dynamic-plugin-sdk';
+
+import useHardwareDevicesPageColumns from './list/hooks/useHardwareDevicesPageColumns ';
+import { HardwareDevicePageRow } from './utils/constants';
+import HardwareDevicesPageRow from './HardwareDevicesPageRow';
+
+type HardwareDevicesPageTableProps = {
+  devices: HardwareDevicePageRow[];
+  error: Error;
+  loaded: boolean;
+};
+
+const HardwareDevicesPageTable: FC<HardwareDevicesPageTableProps> = ({
+  devices,
+  error,
+  loaded,
+}) => {
+  const { t } = useKubevirtTranslation();
+  const columns = useHardwareDevicesPageColumns();
+
+  return (
+    <VirtualizedTable
+      columns={columns}
+      data={devices}
+      EmptyMsg={() => <div>{t('Not available')} </div>}
```

Comment:
div element is unnecessary, change it to Fragment (<></>) and is the space between text and closing tag intentional?

Reply:
The space at the end isn't intentional.
But maybe it's better if I use MutedTextSpan?